table.cc

```diff
@@ -1820,63 +1695,9 @@ future<> table::flush() {
 // be indiscriminately touching the cache during repair. We will just have to
 // invalidate the entries that are relevant to things we already have in the cache.
 future<> table::flush_streaming_mutations(utils::UUID plan_id, dht::partition_range_vector ranges) {
```

Comment:
Why not remove altogether? Who needs this?

Reply:
It is called from the new streaming code too (and ends up just advancing the phaser).

Reply:
I see. Maybe we should rename it to something else, flush is pretty meaningless without the streaming memtables. Can be done in a follow-up.

Reply:
(and rename the `_streaming_flush_phaser` too).